README.md

```diff
@@ -235,6 +297,96 @@ module.exports = {
 };
 ```
 
+### Optimize with [`sharp`](https://github.com/lovell/sharp)
+
+```console
+npm install sharp --save-dev
+```
+
+**Recommended `sharp` options for lossy optimization**
+
+For lossy optimization we recommend using the default settings of `sharp` package.
+The default values and supported file types for each option can be found in the [sharp documentation](https://sharp.pixelplumbing.com/api-output).
+
+**webpack.config.js**
+
+```js
+const ImageMinimizerPlugin = require("image-minimizer-webpack-plugin");
+
+module.exports = {
+  module: {
+    rules: [
+      // You need this, if you are using `import file from "file.ext"`, for `new URL(...)` syntax you don't need it
+      {
+        test: /\.(jpe?g|png)$/i,
+        type: "asset",
+      },
+    ],
+  },
+  optimization: {
+    minimizer: [
+      "...",
+      new ImageMinimizerPlugin({
+        minimizer: {
+          implementation: ImageMinimizerPlugin.sharpMinify,
+          options: {
+            // Your options for `sharp`
+            // https://sharp.pixelplumbing.com/api-output
+          },
+        },
+      }),
+    ],
+  },
+};
+```
+
+**Recommended `sharp` options for lossless optimization**
+
+For lossless optimization we recommend using the options listed below in `minimizer.options.encodeOptions`.
+
+**webpack.config.js**
+
+```js
+const ImageMinimizerPlugin = require("image-minimizer-webpack-plugin");
+
+module.exports = {
+  module: {
+    rules: [
+      // You need this, if you are using `import file from "file.ext"`, for `new URL(...)` syntax you don't need it
+      {
+        test: /\.(jpe?g|png)$/i,
+        type: "asset",
+      },
+    ],
+  },
+  optimization: {
+    minimizer: [
+      new ImageMinimizerPlugin({
+        minimizer: {
+          implementation: ImageMinimizerPlugin.squooshMinify,
+          options: {
+            encodeOptions: {
+              jpeg: {
+                // https://sharp.pixelplumbing.com/api-output#jpeg
+                quality: 100,
+              },
+              webp: {
+                // https://sharp.pixelplumbing.com/api-output#webp
+                lossless: true,
+              },
+              avif: {
+                // https://sharp.pixelplumbing.com/api-output#avif
+                lossless: true,
+              },
```

Comment:
Can we add png/gif here (does sharp support them)? Because it is still popular

Reply:
Sharp supports these formats.
PNG set `quality` by default to 100 which is similar to lossless [doc](https://sharp.pixelplumbing.com/api-output#png)
GIF does not support lossless compression at all [doc](https://sharp.pixelplumbing.com/api-output#gif)

Reply:
Can we add these notes to readme to avoid questions about this?